docs/types/schema-configurations.mdx

```diff
@@ -105,3 +105,23 @@ schema = strawberry.Schema(
     query=Query, config=StrawberryConfig(disable_field_suggestions=True)
 )
 ```
+
+### info_class
+
+By default Strawberry will create an object of type `strawberry.types.Info` when
+the user defines `info: Info` as a parameter to a type or query. You can change
+this behaviour by setting `info_class` to a subclass of `strawberry.types.Info`.
```

Comment:
While `strawberry.type.info` is technically correct, IMHO naming `strawberry.Info` in the docs whill encourage users to subclass the top-level exported `Info` type just like in your example in the release file.

```suggestion
By default Strawberry will create an object of type `strawberry.Info` when
the user defines `info: Info` as a parameter to a type or query. You can change
this behaviour by setting `info_class` to a subclass of `strawberry.Info`.
```

Reply:
Yeah, sounds reasonable, will commit the suggestion.